packs/vsphere/actions/vm_hw_barebones_create.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
Don't need that here since its a module and not a script to be executed in a shell.